lua/entities/gmod_tardis/modules/sh_lock.lua

```diff
@@ -20,7 +20,7 @@ TARDIS:AddControl({
 		text = "Door Lock",
 		pressed_state_from_interior = false,
 		pressed_state_data = "locked",
-		order = 6,
+		order = 10,
```

Comment:
This control appears to have the same order as the door open - not sure if intentional

Reply:
It is intentional. The order of those doesn't matter, they are technically in the same category. What matters is that they're next to each other and after some certain ones. The sorting supports equal order numbers, I've checked that.